### PR TITLE
Merged scripts

### PR DESCRIPTION
**Pull request by turtledreams**
removed the convert_packages script from example
removed its mention from readme
added support to convert example to np version inside the script.py
tested seems to be working fine

### CODE REVIEW

**Comment by turtledreams** [comment]
[[Flutter] Rework the NP script for the example app](https://trello.com/c/YjshQJP2/1054-flutter-rework-the-np-script-for-the-example-app)

File: scripts/script.py [hunk 1]
```diff
@@ -272,6 +306,8 @@ def main():
         modifyFile(modPathCountlySessionsInternal, objectOfComModification, 'mod')
         modifyFile(modPathCountlyUserProfileInternal, objectOfComModification, 'mod')
         modifyFile(modPathCountlyViewsInternal, objectOfComModification, 'mod')
+        modifyFile(modPathExampleYaml, objectOfComModification, 'mod')
+        update_package("../example/lib/", 'package:countly_flutter/', 'package:countly_flutter_np/')
```
Comment (ijunaid) [inline scripts/script.py:310]: I think by adding this line, we can also remove the explicit updating of import/export in SDK files for np.
update_package("../lib/", 'package:countly_flutter/', 'package:countly_flutter_np/')

**Review by ijunaid — CHANGES_REQUESTED** [review]

File: scripts/script.py (hunk 1 above)
Comment (peterBrxwn) [inline scripts/script.py:310]: For consistency, can we use single quotes everywhere?

**Comment by peterBrxwn** [comment]
Please update imports in \integration_test folder

Screenshot 2023-11-08 at 11 15 23

**Comment by peterBrxwn** [comment]
Also, paths to modify does not mention the example folder. Shouldn't it?

`
Paths to modify:
../android/src/main/java/ly/count/dart/countly_flutter/CountlyFlutterPlugin.java
../ios/Classes/CountlyFlutterPlugin.m
../lib/countly_flutter.dart
../lib/countly_state.dart
../lib/countly_config.dart
../lib/remote_config.dart
../lib/remote_config_internal.dart
../lib/sessions_internal.dart
../lib/user_profile_internal.dart
../lib/views_internal.dart
../example/pubspec.yaml
Do you want to continue? (y/n)y
`

**Review by peterBrxwn — CHANGES_REQUESTED** [review]
imports in integration_test folder.

File: scripts/script.py (hunk 1 above)
Comment (turtledreams) [reply scripts/script.py:310]: great point 👍  I currently scan first 30 lines of each file. should we increase that?

Comment (turtledreams) [reply scripts/script.py:310]: indeed

**Comment by turtledreams** [comment]
Okay then I will modify the scrip more to include other things and replace some hard coded parts

File: scripts/script.py (hunk 1 above)
Comment (turtledreams) [reply scripts/script.py:310]: done

**Review by ijunaid — APPROVED** [review]

**Review by peterBrxwn — APPROVED** [review]

**Pull request merged**